### ISSUE TITLE
`testbench-core` and `canvas` are skipped due to `name conflicts`

### ISSUE THREAD

**Issue by bilogic** [orchestral/testbench#424]
- Testbench Version: 8.29.0
- OS: Ubuntu 22.04.5 LTS
- PHP: 8.2.26
- Laravel: 10.48.25
- MySQL: 8.0.37

### Description:
For some reason, testbench and canvas are skipped. And the `vendor/orchestra/testbench` has only 1 composer.json file leading my tests to fail with `Call to undefined function Orchestra\Testbench\laravel_or_fail()`

### Steps To Reproduce:
```
$ composer update
Loading composer repositories with package information
Updating dependencies
Nothing to modify in lock file
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 6 installs, 0 updates, 0 removals
  - Installing orchestra/testbench-core (v8.30.0): Extracting archive
  - Installing orchestra/canvas-core (v8.10.2): Extracting archive
  - Installing orchestra/canvas (v8.12.0): Extracting archive
  - Installing orchestra/workbench (v8.13.0): Extracting archive
  - Installing orchestra/testbench (v8.29.0): Extracting archive
  - Installing orchestra/pest-plugin-testbench (v2.1.0): Extracting archive
 0/6 [>---------------------------]   0%    Skipped installation of bin testbench for package orchestra/testbench-core: name conflicts with an existing file
    Skipped installation of bin canvas for package orchestra/canvas: name conflicts with an existing file
```

**Comment by crynobone** [comment]
1. `Orchestra\Testbench\laravel_or_fail()` is only available on `v9.7.1` and is yet to be available in v7 or v8 https://github.com/orchestral/testbench-core/commit/bbe3b5c23067db564547f88cf4ae33b5b9d61a92
2. Seem like your package also require another package that registers `canvas` which cause the conflict.

If you not using `canvas` you might want to opt to requiring `orchestra/testbench-core` and add any missing deps.

**Comment by bilogic** [comment]
thanks! I found the issue
